packages/sampleProp.conf

```diff
@@ -15,6 +15,10 @@ WebSocketUrl = "Websocket-address-url"
 IotCCUID = "Username"
 IotCCPassword = "Password"
 ShouldUnregisterOnUnload = "False"
+WebsocketCaCert = None
+VerifyCert= True
```

Comment:
If we just change this to False, will the connection stop validating certificate?